astropy/coordinates/matrix_utilities.py

```diff
@@ -136,14 +136,20 @@ def angle_axis(matrix):
     return Angle(angle, u.radian), -axis / r
 
 
-def is_O3(matrix):
+def is_O3(matrix, atol=None):
     """Check whether a matrix is in the length-preserving group O(3).
 
     Parameters
     ----------
     matrix : (..., N, N) array-like
         Must have attribute ``.shape`` and method ``.swapaxes()`` and not error
         when using `~numpy.isclose`.
+    atol : float, optional
+        The allowed absolute difference.
+        Defaulted to ``None``: it is internally initialize depending on the ``dtype``
+        of the matrix.
```

Comment:
Can you keep this in the present tense, maybe something like

```suggestion
        The allowed absolute tolerance. If `None` it defaults to 1e-15 or 5 * epsilon of the matrix's dtype, if floating.
```